plugins/modules/web_infrastructure/nomad_job.py

```diff
@@ -0,0 +1,250 @@
+#!/usr/bin/python
+# coding: utf-8 -*-
+
+# (c) 2020, FERREIRA Christophe <christophe.ferreira@cnaf.fr>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+
+DOCUMENTATION = '''
+---
+module: nomad_job
+author: "FERREIRA Christophe (@chris93111)"
+version_added: "1.2.0"
+short_description: Launch an Nomad Job.
+description:
+    - Launch an Nomad job. See
+      U(https://www.nomadproject.io/api-docs/jobs/) for an overview.
+options:
+    host:
+      description:
+        - FQDN of nomad server.
+      required: true
+      type: str
+    secure:
+      description:
+        - ssl connection.
+      type: bool
+      default: false
+    timeout:
+      description:
+        - timeout of request nomad.
+      type: int
+      default: 5
+    validate_certs:
+      description:
+        - skip validation cert.
+      type: bool
+      default: true
+    cert:
+      description:
+        - path of certificate ssl.
+      type: path
+    key:
+      description:
+        - path of certificate key ssl.
+      type: path
+    namespace:
+      description:
+        - namespace for nomad.
+      type: str
+    token:
+      description:
+        - acl token for authentification.
+      type: str
+    name:
+      description:
+        - name of job for delete or stop job without source.
+      type: str
+    state:
+      description:
+        - type of request
+      choices: ["present", "absent"]
+      required: True
+      type: str
+    force_start:
+      description:
+        - force job to started
+      type: bool
+      default: false
+    source_json:
+      description:
+        - source of json nomad job
+      type: json
+    source_hcl:
+      description:
+        - source of hcl nomad job
+      type: str
+'''
+
+EXAMPLES = '''
+- name: create job
+  nomad_job:
```

Comment:
```suggestion
  community.general.nomad_job:
```
Always use FQCNs in examples.